.github/workflows/spm.yml

```
@@ -73,5 +71,9 @@ jobs:
       run: sudo xcode-select -s /Applications/${{ matrix.xcode }}.app/Contents/Developer
     - name: Setup Scripts Directory
       run: ./setup-scripts.sh
-    - name: Unit Tests
-      run: scripts/third_party/travis/retry.sh scripts/build.sh FirebaseDataConnect ${{ matrix.target }} spm
+    - name: Install iOS Platform
+      run: xcodebuild -downloadPlatform iOS
+    - name: Integration Test Setup
+      run: Tests/Integration/Emulator/start-emulator.sh synchronous
```

Looking at this now, I don't think the arg `synchronous` is even used?
```suggestion
      run: Tests/Integration/Emulator/start-emulator.sh
```

---

Thanks for catching this. Removed it.